### PR TITLE
Request covid certificate screen (EXPOSUREAPP-7426)

### PR DESCRIPTION
**Pull request by mtwalli**
- Creates request GCC consent screen

Testing 
- Testing menu -> Green certificate -> open screen for both types

### CODE REVIEW

File: Corona-Warn-App/src/main/res/values/green_certificate_strings.xml [hunk 1]
```diff
@@ -1,4 +1,29 @@
...
+    <!-- XTXT: Request green certificate title -->
+    <string name="request_green_certificate_title">COVID-19-Testzertifikat</string>
+    <!-- XTXT: Request green certificate subtitle -->
+    <string name="request_green_certificate_subtitle">Sie können über die App ein offizielles digitales Testzertifikat anfordern, das anschließend in der App hinzugefügt wird.</string>
+    <!-- XTXT: Request green certificate body section 1 -->
+    <string name="request_green_certificate_body_1">Das Testzertifikat wird nur bei einem negativen Testergebnis ausgestellt.</string>
+    <!-- XTXT: Request green certificate body section 2 -->
+    <string name="request_green_certificate_body_2">Das Testzertifikat gilt innerhalb der EU als gültiger Nachweis eines negativen Testergebnisses (z.B. für Reisen).</string>
+    <!-- XTXT: Request green certificate body section 3 -->
+    <string name="request_green_certificate_body_3">Das Testzertifikat enhält Daten, die der Prüf-App die Validierung Ihres Zertifikats erlauben.</string>
```
Comment (ChristinaTsigganos) [inline Corona-Warn-App/src/main/res/values/green_certificate_strings.xml:12]: same here, please change "enhält" to "enthält"

**Review by ChristinaTsigganos — CHANGES_REQUESTED** [review]
please fix typo in two sentences, then the PR is okay

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/submission/greencertificate/RequestGreenCertificateFragment.kt [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+            agreeButton.setOnClickListener { viewModel.onAgreeGC() }
+            disagreeButton.setOnClickListener { viewModel.onDisagreeGC() }
+            dateInputEdit.setOnClickListener { openDatePicker() }
+        }
+
+    private fun showDialog() {
+        MaterialAlertDialogBuilder(requireContext())
+            .setTitle(R.string.request_gc_dialog_title)
+            .setMessage(R.string.request_gc_dialog_message)
+            .setNegativeButton(R.string.request_gc_dialog_positive_button) { _, _ -> /* TODO */ }
```
Comment (BMItter) [inline Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/submission/greencertificate/RequestGreenCertificateFragment.kt:55]: negButton ->pos String
posButton ->neg String

File: Corona-Warn-App/src/deviceForTesters/java/de/rki/coronawarnapp/test/greencertificate/GreenCertificateTestFragment.kt [hunk 3]
```diff
@@ -22,13 +25,28 @@ class GreenCertificateTestFragment : Fragment(R.layout.fragment_test_green_certi
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
+
+        binding.pcrScreen.setOnClickListener {
```
Comment (BMItter) [inline Corona-Warn-App/src/deviceForTesters/java/de/rki/coronawarnapp/test/greencertificate/GreenCertificateTestFragment.kt:29]: Binding.apply{...}

**Comment by BMItter** [comment]
looking good overall 👌 small comments

**Review by BMItr — APPROVED** [review]

**Review by ChristinaTsigganos — APPROVED** [review]
UA approved

**Review by jurajkusnier — APPROVED** [review]

**Pull request merged**